theme-cache/web/theme.compiled.css

```diff
@@ -21963,6 +21963,7 @@ html div.dj_ios .widget-switch.auto .widget-switch-btn-wrapper.checked.widget-sw
   margin-left: 8px;
   flex-grow: 0;
   color: #606671;
+  height: 14px;
```

Comment:
Should the theme-cache be committed?

Reply:
There was a previous problem in Studio requiring theme-cache, thats why its not ignored in the .gitignore